lib/c-api/src/wasm_c_api/wasmer.rs

```diff
@@ -0,0 +1,9 @@
+//! Wasmer-specific extensions to the Wasm C API.
```

Comment:
> Should we rename those functions wasmer_…?

possibly, I don't know how much it matters: it probably plays better with others in regards to linking with conditional compilation and all that... but if users are using multiple producers of the Wasm C API in the same binary, then don't they already have that problem?

It would be good to call out wasmer specific functions I suppose.. I'm undecided on this!

> Should we include engine and config functions, and compiler/engine enums too?

I thought about this and decided against it.  The decision was fairly arbitrary but I thought that encapsulating/keeping code near where it's used made more sense than putting it all in one file. I'm open to changing my mind on this or considering alternatives though if you feel strongly about one specific way

Reply:
I don't have a strong opinion on this one. I trust you to take the best decision!